src/lua/compat.h

```diff
@@ -17,6 +17,10 @@ struct lua_State;
 int
 tarantool_lua_compat_init(struct lua_State *L);
 
+/* Toggler for fiber channel graceful close change. */
+int
+fiber_channel_graceful_close_toggle(struct lua_State *L);
```

Comment:
We don't need in the header: a static function in the .c file is enough.

I understand a wish to provide some declaration of provided functions/API description, but I doubt that we should achieve it this way. As for me, `struct luaL_reg` tables works good enough for this.

Reply:
You are right, forgot to remove it like in other options. Fixed.